CONTRIBUTING.md

```diff
@@ -226,83 +208,78 @@ Ensure that you base and target your PR on the `master` branch.
-- Kick off a large round of simulation testing (e.g. 400 seeds for 2k blocks)
-- If errors are found during the simulation testing, commit the fixes to `master`
-  and create a new `RC` branch (making sure to increment the `rcN`)
-- After simulation has successfully completed, create the release branch
-  (`release/vX.XX.X`) from the `RC` branch
-- Create a PR to `master` to incorporate the `CHANGELOG.md` updates
-- Tag the release (use `git tag -a`) and create a release in GitHub
-- Delete the `RC` branches
-
-### Point Release Procedure
-
-At the moment, only a single major release will be supported, so all point releases will be based
-off of that release.
-
-In order to alleviate the burden for a single person to have to cherry-pick and handle merge conflicts
-of all desired backporting PRs to a point release, we instead maintain a living backport branch, where
-all desired features and bug fixes are merged into as separate PRs.
-
-Example:
-
-Current release is `v0.38.4`. We then maintain a (living) branch `sru/release/v0.38.N`, given N as
-the next patch release number (currently `0.38.5`) for the `0.38` release series. As bugs are fixed
-and PRs are merged into `master`, if a contributor wishes the PR to be released as SRU into the
-`v0.38.N` point release, the contributor must:
-
-1. Add `0.38.N-backport` label
-2. Pull latest changes on the desired `sru/release/vX.X.N` branch
-3. Create a 2nd PR merging the respective SRU PR into `sru/release/v0.38.N`
-4. Update the PR's description and ensure it contains the following information:
-   - **[Impact]** Explanation of how the bug affects users or developers.
-   - **[Test Case]** section with detailed instructions on how to reproduce the bug.
-   - **[Regression Potential]** section with a discussion how regressions are most likely to manifest, or might
-     manifest even if it's unlikely, as a result of the change. **It is assumed that any SRU candidate PR is
-     well-tested before it is merged in and has an overall low risk of regression**.
+- Create a new annotated git tag for a release candidate  (eg: `git tag -a v1.1.0-rc1`) in the release branch.
+  - from this point we unfreeze master.
+  - the SDK teams should collaborate and do their best to run test nets in order to validate the release.
+  - once bugs are found, create a PR for `master` and backport to the release branch.
+  - create new release candidate tags once bugs are fixed.
+- Once the team feels the release branch is stable and everything works, create a full release:
+  - update `CHANGELOG.md`.
+  - create a new annotated git tag (eg `git -a v1.1.0`) in the release branch.
+  - Create a GitHub release.
+
+### Minor Release Procedure
+
+A _minor release_ is an increment of the patch number (eg: `v1.2.0` → `v1.2.1`).
 
+Following _semver_ philosophy, we point release after `v1.0` must not break API but they may break consensus.
```

Comment:
```suggestion
Following _semver_ philosophy, point releases after `v1.0`:

 - must not break API
 - can break consensus
```

Reply:
I think we're saying that it's permissible for consensus break? in this is true, let's use "can"

Reply:
may
In general, reserve for official policy or legal considerations. To convey possibility, use can or might instead. **To convey permission, use can instead.**

Reply:
> I think we're saying that it's permissible for consensus break?

yes